src/zope/mkzeoinstance/tests/test_unix.py

```diff
@@ -12,6 +12,8 @@
 #
 ##############################################################################
 
+# Run Tests: python -m unittest test_unix
+
```

Comment:
While this may work with some packages it is not the canonical way unit tests are run for zopefoundation packages, we use `zope.testrunner` in conjunction with `tox` by default. Please remove this comment.

Reply:
comment removed.